src/kOS/Suffixed/BodyTarget.cs

```diff
@@ -96,6 +96,7 @@ private void BodyInitializeSuffixes()
             AddSuffix("ROTATIONPERIOD", new Suffix<double>(() => Body.rotationPeriod));
             AddSuffix("ATM", new Suffix<BodyAtmosphere>(() => new BodyAtmosphere(Body)));
             AddSuffix("ANGULARVEL", new Suffix<Vector>(() => RawAngularVelFromRelative(Body.angularVelocity)));
+            AddSuffix("SOIRADIUS", new Suffix<double>(() => Body.sphereOfInfluence));
```

Comment:
@abenkovskii have you tested this in game? i thought i remembered this not being a super reliable property of the body class but squad may have fixed it.